README.md

```diff
@@ -3,5 +3,12 @@
 A bash script to monitor a RaspiBolt node health and activity at a glance.
 
 ## Installation
+`cd /tmp`
+
+`git clone https://github.com/raspibolt/raspibolt-pulse`
+
+`sudo cp *.sh /usr/local/bin/`
+
+`sudo mv /usr/local/bin/raspibolt-pulse-switch.sh /usr/local/bin/raspibolt`
 
 To use this script, follow the following RaspiBolt [bonus guide](https://raspibolt.org/guide/bonus/raspberry-pi/system-overview.html).
```

Comment:
We will need to change the instructions in the system-overview guide:

1. Change the name of the guide itself to RaspiBolt Pulse
2. Explains how to make the all the scripts executable
3. Explains that the user must edit the main script to choose the following options:
** LND vs CLN (LND default)
** eth0 vs wlan0 (wlan0 default, so far..)
** and later on possibly other choices as well as we expand the modularity of the script